.github/workflows/fossa-workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: fossa-workflow
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+########## FOSSA-ANALYZE ##########
+
+  fosa-analyze:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: |
+          echo "MAGMA_ROOT=/tmp/magma" >> $GITHUB_ENV
```

Comment:
instead of defining MAGMA_ROOT like this, we should use the github predefined variable
```
    env:
      MAGMA_ROOT: "${{ github.workspace }}"
```

Reply:
agree